src/desktop/apps/auction/components/layout/auction_info/Registration.js

```diff
@@ -29,37 +30,57 @@ function Registration(props) {
         } else if (!isQualifiedForBidding) {
           return (
             <div className={b("wrapper")}>
-              <button className="avant-garde-button-black is-block is-disabled">
+              <Button width="100%" size="large" isDisabled>
                 Registration pending
-              </button>
-              <div className={b("small", { warning: true })}>
+              </Button>
+              <Sans mt="1" size="3" color="yellow100" textAlign="center">
                 Reviewing submitted information
-              </div>
+              </Sans>
             </div>
           )
         } else if (numBidders > 0) {
           return (
             <div className={b("approved")}>
               <span className="icon-check" />
-              Approved to Bid
+              <Sans mt="1" size="3" color="black60" textAlign="center">
+                Approved to Bid
```

Comment:
I think we still want the `Approved to Bid` text to be green, so we may not want this just for now.

Reply:
Yes, I broke this. Fixing now.

Reply:
<img width="221" alt="Screen Shot 2020-04-13 at 3 56 44 PM" src="https://user-images.githubusercontent.com/19369/79155621-6706ea80-7d9f-11ea-89b6-b7c85fd1e8e1.png">